examples/pypi-bdist-packages/pixi.toml

```diff
@@ -0,0 +1,14 @@
+[project]
+channels = ["conda-forge"]
+name = "pypi-bdist-packages"
+platforms = ["osx-arm64", "osx-64", "linux-aarch64", "linux-64", "win-64"]
+
+[dependencies]
+python = "3.11.*"
+pip = ">=25.0.1,<26"
+
+[pypi-dependencies]
+numpy = ">=2.2.3, <3"
+# torch has local version, which is percent encoded on url, it should be decode.
+# torch-2.6.0%2Bcpu + torch-2.6.0+cpu
+torch = { version = ">=2.2", index = "https://download.pytorch.org/whl/cpu" }
```

Comment:
I only know that this package has a local version, but it is big (170MB). If there is a smaller package. We can change it.

Reply:
for the general unit tests, i would prefer a very small pypi package like [ordered-enum](https://pypi.org/project/ordered-enum/) which is a wheel with zero dependencies that is only 3KB large.

how about you verify the behavior with URL encoding in a unit test in `pack.rs` `test_url_encoding` where you just call `download_pypi_package` with the specific pytorch url?
this way pytorch will only get downloaded a single time in that test and all other integration tests just use a very small pixi environment which should make the tests quicker